### DISCUSSION TITLE
Custom metrics from zabbix to skywalking

### DISCUSSION THREAD

**Discussion by pramod-desai-minutus** [apache/skywalking#10327] [Q&A]
Hi,

I have configured zabbix agent to send standard metrics for skywalking UI.

How to send custom metrcis from zabbix to skywalking.

Any document ,link will help me .

Thanks in advance!!

**Comment by wu-sheng** [comment]
Send any metric you want(from zabbix setup) We don't have doc for it. 
At oap side, you need to learn MAL and set up UI.

**Reply by pramod-desai-minutus** [reply]
Thank you!
